lib/CArtHandler.cpp

```diff
@@ -851,15 +856,15 @@ void CArtifactInstance::putAt(ArtifactLocation al)
 	assert(canBePutAt(al));
 
 	al.getHolderArtSet()->setNewArtSlot(al.slot, this, false);
-	if(!ArtifactUtils::isSlotBackpack(al.slot))
+	if(!ArtifactUtils::isSlotBackpack(al.slot) && (al.slot != ArtifactPosition::TRANSITION_POS))

```

Comment:
Perhaps add another method similar to isSlotBackpack, e.g. isSlotEquipment (which returns true for all slots that are considered as "equipped" and provide bonuses to hero)

Reply:
done